support/ruby/collins-state/collins_state.gemspec

```diff
@@ -32,22 +27,17 @@ Gem::Specification.new do |s|
   s.homepage = "https://github.com/tumblr/collins/tree/master/support/ruby/collins-state"
   s.licenses = ["APL 2.0"]
   s.require_paths = ["lib"]
-  s.rubygems_version = "1.8.23"
   s.summary = "Collins based state management"
 
-  if s.respond_to? :specification_version then
-    s.specification_version = 3
+  s.add_runtime_dependency 'collins_client'
```

Comment:
Should we add some more pessimistic version dependencies maybe? At least have the runtime dependencies pinned to their current minor version?